build.gradle.kts

```diff
@@ -143,3 +143,9 @@ val enableJVMFrontend: Boolean by extra {
     enableJVMFrontend.toBoolean()
 }
 project.logger.lifecycle("JVM frontend is ${if (enableJVMFrontend) "enabled" else "disabled"}")
+
+val enableConfigfilesFrontend: Boolean by extra {
```

Comment:
-> `enableConfigFilesFrontend` ?

Reply:
I changed this to `enableINIFrontend` to be in line with the review changes above.